tempodb/encoding/streaming_block_test.go

```diff
@@ -59,7 +59,8 @@ func TestStreamingBlockAddObject(t *testing.T) {
 
 	numObjects := (rand.Int() % 20) + 1
 	cb, err := NewStreamingBlock(&BlockConfig{
-		BloomFP:              .01,
+		BloomFP:              0.01,
+		BloomFilterShardSize: 100_000,
```

Comment:
let's make this much smaller. 1000? 10_000? just to make sure we're getting multiple shards.